indico/modules/events/contributions/models/contributions.py

```diff
@@ -295,6 +296,14 @@ def speakers(self):
         return [person_link for person_link in self.person_links if person_link.is_speaker]
 
     @property
+    def primary_authors(self):
```

Comment:
@mic4ael added properties for those.. maybe cherrypick his commit?

f92dddd11f2176021f25dde1bc0bb5d20bc7a7b7